libcodechecker/libhandlers/server.py

```diff
@@ -342,6 +369,193 @@ def arg_match(options):
     parser.set_defaults(func=__handle)
 
 
+def handle_connect_ret(ret):
+    """
+    Print out a message based on the database statuses.
+    """
+    if ret == DBStatus.OK:
+        return "Database is OK."
+    elif ret == DBStatus.FAILED_TO_CONNECT:
+        return "Failed to connect to the configured database"
+    elif ret == DBStatus.SCHEMA_INITIALIZATION_ERROR:
+        return "Schema initialization error"
+    elif ret == DBStatus.SCHEMA_MISSING:
+        return "Schema is missing"
+    elif ret == DBStatus.SCHEMA_MISMATCH_OK_MIGRATION:
+        return "Schema mismatch"
+    elif ret == DBStatus.SCHEMA_MISMATCH_NO_MIGRATION:
+        return "Automatic migration is not available"
+    elif ret == DBStatus.MISSING:
+        return "Database is missing"
+
+
+def get_user_input(msg):
+    """
+    Get the user input.
+
+    :returns: True/False based on the asnwer from the user
+    """
+    yes = ['y', 'Y', 'yes', 'Yes']
```

Comment:
It would be better to convert the user input to lower case and check that input. In this case we can handle the YES case too.
```py
yes = ['y', 'yes']
answer = raw_input(msg)
if answer.lower() in yes:
    return True
return False
```